wormhole-connect/src/config/types.ts

```diff
@@ -64,7 +64,7 @@ export enum TokenIcon {
   'XLAYER',
   'MANTLE',
   'WORLDCHAIN',
-  'UNICHAIN',
+  'BERA',
```

Comment:
why'd you remove `UNICHAIN` here?

Reply:
The gas token on Unichain is ETH and so Connect uses the Ethereum icon when displaying the Uniswap gas token. TokenIcon.UNICHAIN is not used so I just removed it